pygments/lexers/configs.py

```diff
@@ -1127,3 +1127,28 @@ def analyse_text(text):
             result += 0.49
 
         return result
+
+
+class UnixConfigLexer(RegexLexer):
+    """
+    Lexer for Unix/Linux config files using colon-separated values, e.g.
+        * ``/etc/group``
+        * ``/etc/passwd``
+        * ``/etc/shadow``
+    
```

Comment:
This should be dedented and separated from the previous paragraph by a blank line in order not to be interpreted as a block quote (reST syntax quirks).